haystack/nodes/reader/farm.py

```diff
@@ -1248,7 +1226,7 @@ def predict_on_texts(self, question: str, texts: List[str], top_k: Optional[int]
          ```
 
         :param question: Question string
-        :param documents: List of documents as string type
+        :param texts: List of documents as string type
```

Comment:
As above, I wouldn't call it "documents" here as they have their own type. Let's call it texts instead or maybe document texts.